tools-v2/internal/proto/curvebs/schedule/statuscode.proto

```diff
@@ -0,0 +1,30 @@
+/*
+ *  Copyright (c) 2023 NetEase Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+/*
+* Project: curvecli
+* Created Date: 2023-06-14
+* Author: montaguelhz
+ */
+
+syntax="proto2";
+option go_package = "proto/schedule/statuscode";
+
+enum SchStatusCode {
```

Comment:
full spell is better(ScheduleStatusCode).

Reply:
done